source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnRenderer.mm

```diff
@@ -113,4 +115,26 @@ - (void)configUpdateForUIImageView:(ACRView *)rootView acoElem:(ACOBaseCardEleme
     renderBackgroundImage(rootView, backgroundImageProperties.get(), imageView, image);
 }
 
+- (void)configureBorderForElement:(ACOBaseCardElement *)acoElem container:(ACRContentStackView *)container config:(ACOHostConfig *)acoConfig
+{
+    std::shared_ptr<BaseCardElement> elem = [acoElem element];
+    std::shared_ptr<Column> containerElem = std::dynamic_pointer_cast<Column>(elem);
+    bool shouldShowBorder = containerElem->GetShowBorder();
+    if(shouldShowBorder)
+    {
+        container.layer.borderWidth = 1;
```

Comment:
Should borderWidth(1pt) and cornerRadius(5pt) come from host config? Or any central constant file needed?
@ardlank @GabrielMedAlv

Reply:
I think could be a good idea to have them on host config, do you agree @ardlank, to have it on android on host config too?

Reply:
Yes, there should be a default value and the property can be overridden by host config. @wyAbhishek @GabrielMedAlv

Reply:
Looks like there is an issue with this.

1. We already have borderThickness with default value 0 [here](https://github.com/microsoft/AdaptiveCards-Mobile/blob/a359afded0f946a5c386b81c94a065d59157869f/source/shared/cpp/ObjectModel/HostConfig.h#L199C1-L200C1).  If we change default value then all renderers using ACRContentView will start showing border. 
2. Image can't have this because image doesn't have container style and rounded corner is based on ImageStyle.
3. From the Code comment [here](https://github.com/microsoft/AdaptiveCards-Mobile/blob/a359afded0f946a5c386b81c94a065d59157869f/source/shared/cpp/ObjectModel/HostConfig.h#L199C1-L200C1), it seems that borderThickness is not consumed in Android. 

IMO, we should decide clear schema for HostConfig and then move constants accordingly. I can do that in follow up PRs if we decide to change anything.